ocaml/jane/doc/extensions/unboxed-types/index.md

```diff
@@ -125,6 +131,7 @@ Each numeric type has its own library for working with it: `float_u`,
 modules in the `janestreet_shims` library.)
 
 * Unboxed constants are written with a prepended `#`.
+  There is no literal syntax for unboxed vectors: use the `Ocaml_simd_sse` library instead.
```

Comment:
Is our plan to public release this library on the bleeding-edge opam repo?  (Maybe make a ticket somewhere to note that plan if so, supposing one doesn't exist).